config/file_test.go

```diff
@@ -214,17 +214,17 @@ func TestFileStoreNew(t *testing.T) {
 	t.Run("relative path, file does not exist", func(t *testing.T) {
 		os.Clearenv()
 
-		err := os.MkdirAll("TestFileStoreNew/a/b/c", 0700)
+		err := os.MkdirAll("config/TestFileStoreNew/a/b/c", 0700)
```

Comment:
Should call to `NewMainHelper` default to setting up everything?

Reply:
Yeah, let's preserve the existing behaviour just until we can get enterprise over the line.

Reply:
@lieut-data `EnableResources` should be true only when we`re creating an App?
Actually while setting up resources, apart from copying config we are copying/symlinking other resources as well. Since tests run with different cwd which if don't have required resources would cause tests to fail.

Reply:
Actually this change would still be needed even if we do not setup default config in temp since call to `config.NewFileStore(<path>, <bool>)` resolves to `config/<path>` if config at path does not exists.

Reply:
I /think/ all of those resources are only used for `App` (except maybe the weird .json edge case you had to handle, but I feel like that test pattern is invalid.) So presumably we only need resources setup then?

With respect to this particular test, the test passes today without rooting in `config/`, right? Why would it start failing with your changes?

Reply:
I need to check usage of test resources in packages other that packages creating an App.

This particular test is failing with my changes since it now runs with pwd set to parent of config folder while without my changes they run from config/, which is path of the package whos tests are being run. When this test runs with pwd set to config/, everything falls in place and it works.

Reply:
Ok, so I think this boils back to the need to disable resources for this test: when PWD is switched to the temporary folder, but we disable resource copy and don't pre-create a `config` folder beneath it, it should no longer fall back to looking for a `config/` path, right?

Reply:
Yes, thats right. It would work same as it worked previously without my changes

Reply:
Ok so lets disable resource setup for this test. What should be criteria for disabling resource setup for other packages, should it be not creating app and not using config files?

Reply:
Yeah, that would be my gut instinct -- in fact, probably any test also not setting up the store :)